packages/graph/hash_graph/lib/graph/src/store/postgres/ontology/read.rs

```diff
@@ -50,15 +54,16 @@ where
 }
 
 #[async_trait]
-impl<C: AsClient, T> crud::Read<'_, (), T> for PostgresStore<C>
+impl<C: AsClient, T> crud::Read<'_, AllLatest, T> for PostgresStore<C>
```

Comment:
Can we maybe put a comment above this saying that this impl is an example of why we don't like our current design and intend to move towards something more like a builder / AST approach

Reply:
Added a comment on the `Read` trait: 8ac6485ed35a53b030e15c892dcb63bd539bf4f8